app/(blobs)/(setup)/_components/OnboardSteps/Documentation.tsx

```diff
@@ -1,18 +1,28 @@
+import { createId } from '@paralleldrive/cuid2';
 import { FileText } from 'lucide-react';
-import { setAppConfigured } from '~/actions/appSettings';
+import { redirect } from 'next/navigation';
+import { setAppSetting } from '~/actions/appSettings';
 import Section from '~/components/layout/Section';
 import { Button } from '~/components/ui/Button';
 import SubmitButton from '~/components/ui/SubmitButton';
 import Heading from '~/components/ui/typography/Heading';
 import Paragraph from '~/components/ui/typography/Paragraph';
+import { env } from '~/env';
 import trackEvent from '~/lib/analytics';
 
 function Documentation() {
   const handleAppConfigured = async () => {
-    await setAppConfigured();
+    const installationId = env.INSTALLATION_ID ?? createId();
```

Comment:
This is returning undefined even with an installationId in the env. Needs investigation.